grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -943,7 +943,7 @@ class ReportService implements ApplicationContextAware {
                                invoice.invoice_type_id = :prepaymentInvoiceId
                                    OR invoice.invoice_type_id IS NULL
                                    OR invoice.date_posted IS NULL
-                                   OR (invoice.date_posted IS NOT NULL AND adjustment_invoice_amount.total_invoiced_amount < order_adjustment_details.total_adjustment)
+                                   OR (invoice.date_posted IS NOT NULL AND ABS(adjustment_invoice_amount.total_invoiced_amount) != order_adjustment_details.total_adjustment)
```

Comment:
@drodzewicz Shouldn't this part `order_adjustment_details.total_adjustment` be abs as well?

Reply:
it is already ABS'ed in the `order_adjustment_details` view